integreat_cms/cms/templates/_related_contents_table.html

```diff
@@ -0,0 +1,65 @@
+    </div>
+    <table class="w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white">
+        <thead>
+            <tr class="border-b border-solid border-gray-200">
+                <th class="text-sm text-left uppercase py-3 pl-4 pr-2">
+                    {% translate "Name in " %} {{ region_default_language.translated_name }}
+                </th>
+                {% if region_default_language != backend_language %}
+                    <th class="text-sm text-left uppercase py-3 pl-4 pr-2">
+                        {% translate "Name in " %} {{ backend_language.translated_name }}
+                    </th>
+                {% endif %}
+            </tr>
+        </thead>
+        <tbody>
+            {% for content in contents %}
+                {% get_translation content region_default_language.slug as content_translation %}
+                {% get_translation content backend_language.slug as backendlang_content_translation %}
+                <tr class="border-b border-solid border-gray-200 hover:bg-gray-100">
+                    {% if content_translation %}
+                        <td class="text-sm text-left py-3 pl-4 pr-2">
+                            <a href="{{ content_translation.backend_edit_link }}"
+                               class="block py-1.5 px-2 overflow-hidden max-w-xs whitespace-nowrap text-ellipsis text-gray-800"
+                               title="{% translate "View content" %}">
+                                {{ content_translation.title }}
+                            </a>
+                        </td>
+                    {% else %}
+                        <td class="text-sm text-left py-3 pl-4 pr-2">
+                            <i>{% translate "Translation not available" %}</i>
+                        </td>
+                    {% endif %}
+                    {% if region_default_language != backend_language %}
+                        {% if backendlang_content_translation %}
+                            <td class="text-sm text-left py-3 pl-4 pr-2">
+                                <a href="{{ backendlang_content_translation.backend_edit_link }}"
+                                   class="block py-1.5 px-2 overflow-hidden max-w-xs whitespace-nowrap text-ellipsis text-gray-800"
+                                   title="{% translate "View content" %}">
+                                    {{ backendlang_content_translation.title }}
+                                </a>
+                            </td>
+                        {% else %}
+                            <td class="text-sm text-left py-3 pl-4 pr-2">
+                                <i>{% translate "Translation not available" %}</i>
+                            </td>
+                        {% endif %}
+                    {% endif %}
+                </tr>
+            {% empty %}
+                <tr>
+                    <td colspan="2" class="px-4 py-3">
+                        {% trans no_content_message %}
```

Comment:
Somehow this won't be translated into German, while `{% translate content_type_plural|title %}` with either `content_type_plural="pages` or `content_type_plural="locations` are working 🤔

Reply:
The problem appears to be that the translation strings for "This organization currently has no maintained pages." and "This organization currently has no maintained locations." are not defined. You can even see in the diff that the translation tool has removed the existing translation. This is probably because it does not recognize that those two message should be translated anymore. I guess the easiest fix is to remove the `{% trans … %}` block here and translate `no_content_message`  already before passing it to `_related_contents_table.html`
TIL django supports using [this](https://docs.djangoproject.com/en/5.1/topics/i18n/translation/#string-literals-passed-to-tags-and-filters) syntax for that:
```python
{% include "../_related_contents_table.html" with contents=form.instance.pages.all content_type_plural="pages" no_content_message=_("This organization currently has no maintained pages.") %}
{% include "../_related_contents_table.html" with contents=form.instance.pois.all content_type_plural="locations" no_content_message=_("This organization currently has no maintained locations.") %}
```

Reply:
@david-venhoff Thank you so much 🙏 It's working now 😸